cosmos/precompile/governance/governance_test.go

```diff
@@ -176,31 +117,32 @@ var _ = Describe("Governance Precompile", func() {
 				ToAddress:   caller.String(),
 				Amount:      initDeposit,
 			}
-
 			metadata := "metadata"
 			title := "title"
 			summary := "summary"
-
-			msg, err := codectypes.NewAnyWithValue(message)
+			msgBz, err := message.Marshal()
 			Expect(err).ToNot(HaveOccurred())
-
+			// Create and marshal the proposal.
+			proposal := v1.MsgSubmitProposal{
+				// Messages:       []*codectypes.Any{msg},
```

Comment:
no messages?

Reply:
proposal and msgs are diff params put into the method, removed the comment, proposalBz is on line 124